### DISCUSSION TITLE
Welcome to Plotly.Blazor Discussions!

### DISCUSSION THREAD

**Discussion by sean-mcl** [LayTec-AG/Plotly.Blazor#64] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by alsuomi** [comment]
Hey

After upgrading the nuget package to 2.0.0, it has stopped working.
The references cannot be found even if the Nuget Package's installation is successful.

Can't even downgrade to 1.5.0!!

- The name 'ModeFlag' does not exist in the current context
- The type or namespace name 'Plotly' could not be found (are you missing a using directive or an assembly reference?)
- The type or namespace name 'Layout' could not be found (are you missing a using directive or an assembly reference?
- The type or namespace name 'ITrace' could not be found (are you missing a using directive or an assembly reference?)
- The type or namespace name 'Config' could not be found (are you missing a using directive or an assembly reference?)
- The type or namespace name 'PlotlyChart' could not be found (are you missing a using directive or an assembly reference?)
- The type or namespace name 'Scatter' could not be found (are you missing a using directive or an assembly reference?)

Plugin version 1.5.0 is working fine with a NEW blazor project. However, as I mentioned, I can't downgrade anymore.

Please reply and provide some support what to do after the update.

A quick support will be appreciated. Thanks!
